### DISCUSSION TITLE
与AntiBV脚本共同使用时注意关闭网址参数清理

### DISCUSSION THREAD

**Discussion by MasterOfStar** [the1812/Bilibili-Evolved#2691] [分享 / Share]
如果在使用本脚本同时使用 [Bilibili AntiBV 脚本](https://greasyfork.org/zh-CN/scripts/398499-bilibili-antibv) 
需要关闭网址参数清理功能
![网址参数清理](https://user-images.githubusercontent.com/21274896/145723267-f5e05be6-ceb9-4d69-a71e-517e7a350c45.png)

否则可能导致默认宽屏失效（反正我不能完全确定就是这个的问题，所以没有去提issue）

**Comment by MasterOfStar** [comment]
对了 是旧版播放器页面，不是新版。
（另外在仅使用本脚本情况下，合集内容手动换集的时候不会自动宽屏是什么鬼）

**Reply by FoundTheWOUT** [reply]
#2408
